src/psyclone/psyir/frontend/fparser2.py

```diff
@@ -2900,7 +3014,7 @@ def _part_ref_handler(self, node, parent):
 
         '''
         reference_name = node.items[0].string.lower()
-        symbol = parent.find_or_create_symbol(reference_name)
+        symbol = _find_or_create_imported_symbol(parent, reference_name)
 
         array = ArrayReference(symbol, parent)
```

Comment:
This file has a couple of pycodestyle indentation errors.

Reply:
Done